### PR TITLE
Add default stream codec for format

### PR DESCRIPTION
**Pull request by kice**
Get default output codec from the output format.

### CODE REVIEW

**Comment by jlaine** [comment]
Hi! Would you mind fixing the linter error please?

File: .gitignore [hunk 1]
```diff
@@ -15,6 +15,8 @@
 *.egg-info
 *.pyc
 *.so
+*.lib
+*.dll
```
Comment (jlaine) [inline .gitignore:19]: This change seems unrelated to the PR

Comment (kice) [reply .gitignore:19]: I could remove it. 
Since I wrote the code under windows, without this, it will just be a little bit painful to make a commit.

Comment (jlaine) [reply .gitignore:19]: Yes kindly remove this change and rebase on top of `develop`, I added these files in f8cb3a352015c24b05de01a228d2cde54ae71a64

**Comment by WyattBlue** [comment]
This PR lacks justification given its lines of code added. This line especially makes me doubt its quality. `libx264` is an *encoder*, not a codec.
```
self.assertEqual(output.format.video_codec.name, 'libx264')
```

Due to this, and age / merge conflict issues, I'm going to close this PR.
